### ISSUE TITLE
WP crash on update on PHP version ">= 7.2.5"

### ISSUE THREAD

**Issue by PsychoBitchy** [patrickposner/simply-static#1]
"Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 7.2.5". "
WP now crashes after plugin update ( last version 2.1.2 ) on a server where PHP version is < 7.2.5
...\plugins\simply-static\vendor\composer\platform_check.php
```
if (!(PHP_VERSION_ID >= 70205)) {
    $issues[] = 'Your Composer dependencies require a PHP version ">= 7.2.5". You are running ' . PHP_VERSION . '.';
}
```
However in readme.txt:
```
=== Simply Static ===
Contributors: patrickposner
Tags: html, static website generator, static site, secure, fast
Requires at least: 5.4
Tested up to: 5.6
**Requires PHP: 5.6**
Stable tag: 2.1.2
License: GPLv2 or later
License URI: http://www.gnu.org/licenses/gpl-2.0.html
```

**Comment by patrickposner** [comment]
@PsychoBitchy thanks for mentioning it, I also noticed the inconsistency here.
I modify it later today, the new minimum version of PHP will (and should) be 7.2.5.

**Comment by patrickposner** [comment]
This is fixed now in 5e0eb962abf5bd4a6a4c5e88fb270569b76948ab

**Comment by PsychoBitchy** [comment]
Thanks!
